src/Forms/ChangeCasing.cs

```diff
@@ -65,13 +67,15 @@ internal void FixCasing(Subtitle subtitle, string language)
             // Longer names must be first
             namesEtc.Sort((s1, s2) => s2.Length.CompareTo(s1.Length));
 
+            bool fixLowercaseI = language.StartsWith("en", StringComparison.Ordinal) && radioButtonNormal.Checked;
+
             string lastLine = string.Empty;
             foreach (Paragraph p in subtitle.Paragraphs)
             {
                 p.Text = FixCasing(p.Text, lastLine, namesEtc);
 
                 // fix casing of English alone i to I
-                if (radioButtonNormal.Checked && language.StartsWith("en", StringComparison.Ordinal))
+                if (fixLowercaseI)

```

Comment:
Ah no-no... "heavy" may be wrong word in this case I just mean the whole operation can be reduced by 1...